### PR TITLE
Issue-#451: Fixed last few "style={{ }}"s

### PR DESCRIPTION
**Pull request by Ralphvandodewaard**
All use of "style={{ }}" has been corrected now.

### CODE REVIEW

File: src/list-item/index.jsx [hunk 1]
```diff
@@ -146,7 +146,7 @@ class ListItem extends Component {
     }
 
     return (
-      <div style={{ width: '100%' }}>
+      <div>
```
Comment (larstadema) [inline src/list-item/index.jsx]: You removed this property. I am guessing this was done on purpose. Please create a style object with this inside. Also create an comment to ask @sjaakluthart  or @IanCStewart if this is still needed here.

Comment (Ralphvandodewaard) [inline src/list-item/index.jsx]: No style object was made for this property because the style didn't have any effect on the div element.

Comment (IanCStewart) [inline src/list-item/index.jsx]: This must've been for something. Looks like @sjaakluthart added this so check with him or this is oke.

Comment (sjaakluthart) [inline src/list-item/index.jsx]: I've added this to make sure ListItems don't break in a flex box, this should not be removed.

**Pull request merged**